appveyor_before_build.ps1

```diff
@@ -0,0 +1,18 @@
+. ./appveyor_helpers.ps1
+
+function PatchCsharpPlugin
+{
+    Add-AppveyorMessage -Message "Copying the C# plugin for patching"
+        
+    $csPluginCleanJar = FindSingleFile "$mavenLocalRepository\org\sonarsource\dotnet\sonar-csharp-plugin" "*SNAPSHOT.jar"
+   
+    $destinationFile = [System.IO.Path]::Combine($env:APPVEYOR_BUILD_FOLDER, "PackagingProjects\CSharpPluginPayload", "csharp_plugin.jar");
+    [System.IO.File]::Copy($csPluginCleanJar, $destinationFile)
+}
+
+
+#Add-AppveyorMessage -Message "Building the latest working C# plugin"
```

Comment:
I will uncomment this. Add-AppveyorMessage is an AppVeyor API that logs text to a separate messages tab.